File: pinyougou-web-portal/src/main/webapp/js/service/contentService.js
app.service("contentService",function($http){
	this.findByCategoryId = function(categoryId){
		return $http.get("content/findByCategoryId.do?categoryId="+categoryId);
	};

    this.findByGoods = function(){
        return $http.get("content/findByGoods.do");
    };

    this.findItemByCat = function(parentId){
        return $http.get("content/findItemByCat.do?parentId="+parentId);
    };

});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/ContentService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.ad.Content;
import cn.itcast.core.pojo.item.ItemCat;
import entity.PageResult;

import java.util.List;

public interface ContentService {

	public List<Content> findAll();
	
	public PageResult findPage(Content content, Integer pageNum, Integer pageSize);
	
	public void add(Content content);
	
	public void edit(Content content);
	
	public Content findOne(Long id);
	
	public void delAll(Long[] ids);

    List<Content> findByCategoryId(Long categoryId);

	List<Object> findByGoods();

	List<ItemCat> findItemByCat(Long parentId);
}
